### PR TITLE
Disables the element form inputs on graded questions

### PR DESCRIPTION
**Pull request by trombonekenny**
(Actually implements data['editable'] for MC and checkbox questions

And my first v3/python contribution to PL

### CODE REVIEW

File: elements/pl_checkbox/pl_checkbox.py [hunk 1]
```diff
@@ -94,16 +94,18 @@ def render(element_html, element_index, data):
     correct_keys = [answer['key'] for answer in correct_answer_list]
 
     if data['panel'] == 'question':
-        editable = data['editable']
         partial_score = data['partial_scores'].get(name, {'score': None})
         score = partial_score.get('score', None)
 
+        if (score is not None):
```
Comment (jakebailey) [inline elements/pl_checkbox/pl_checkbox.py]: No prens in Python, please. :P

File: elements/pl_multiple_choice/pl_multiple_choice.py [hunk 2]
```diff
@@ -81,16 +81,18 @@ def render(element_html, element_index, data):
     correct_key = data['correct_answers'].get(name, {'key': None}).get('key', None)
 
     if data['panel'] == 'question':
-        editable = data['editable']
+
         partial_score = data['partial_scores'].get(name, {'score': None})
         score = partial_score.get('score', None)
+        if (score is not None):
```
Comment (jakebailey) [inline elements/pl_multiple_choice/pl_multiple_choice.py]: Same here, no prens.

File: elements/pl_checkbox/pl_checkbox.py (hunk 1 above)
Comment (trombonekenny) [reply elements/pl_checkbox/pl_checkbox.py]: Cite the reason please?

Comment (trombonekenny) [reply elements/pl_checkbox/pl_checkbox.py]: Oh I see the PEP8 reference above. I'll read up.

Comment (jakebailey) [reply elements/pl_checkbox/pl_checkbox.py]: Irritatingly the actual PEP8 document doesn't say not to do it explicitly, but generally you shouldn't unless you're trying to deal with multiline statements, like in:

https://www.python.org/dev/peps/pep-0008/#id17

If you use an editor or tool that tells you about python style, it'd say to remove them, anyway.

**Comment by mwest1066** [comment]
What problem is this PR fixing? It breaks MC questions on exams with multiple attempts, because after the first attempt it doesn't allow further editing of the answer.

**Comment by trombonekenny** [comment]
I noticed when looking into the MC bug last week that 'editable' support was started (really just a variable made) but never implemented. The small cosmetic thing it fixes is disabling the inputs so you can't change them after a submission. The more valuable fix is me learning more about PL development and question elements. :)

**Comment by mwest1066** [comment]
This got implemented elsewhere along the way.

**Pull request closed without merging**